pkg/playground/speed.html

```diff
@@ -76,7 +76,7 @@ <h1>Download Test</h1>
             <tbody>
                 <tr>
                     <td><label class="control-label" for="message">Path</label></td>
-                    <td><input class="form-control" id="download-path" value="/dev/sda"></td>
+                    <td><input class="form-control" id="download-path" value="/etc/os-release"></td>
```

Comment:
/dev/sda wasn't ideal for sure, as normal users can't read it. But /etc/os-release doesn't provide much of a speed test. I suppose this is a debugging leftover?

Reply:
Ah, I wasn't thinking of testing speed at all.  I revert this.

Reply:
Done